src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -383,7 +383,9 @@ public ColumnFunctionAccessor(ImmutableArray<ColumnOptions> infos)
         [BestFriend]
         internal readonly IReadOnlyList<IColumnFunction> ColumnFunctions;
 
-        public readonly ImmutableArray<ColumnOptions> Columns;
+        [BestFriend]
+        internal readonly ImmutableArray<ColumnOptions> Columns;
```

Comment:
>ImmutableArray [](start = 26, length = 14)

@rogancarr I've ask @wschin to hide this right now, for a reason we don't have common idea of how to do it. Hope you Ok with that. We can always expose it in v1.1

Reply:
we need to leave this public for this transform, as there is a user use-case. 
Or we need to come up with another way to expose the model parameters for the normalizer, but we cannot hide them. 



---
In reply to: [264910410](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264910410) [](ancestors = 264910410)

Reply:
This field is public now.

---
In reply to: [265306118](https://github.com/dotnet/machinelearning/pull/2888#discussion_r265306118) [](ancestors = 265306118,264910410)

Reply:
If you can add a comment about not changing this back to internal, as it has customer usage. 

I'll add a functional test about it 

---
In reply to: [265317859](https://github.com/dotnet/machinelearning/pull/2888#discussion_r265317859) [](ancestors = 265317859,265306118,264910410)

Reply:
It has a test added by Rogan.

---
In reply to: [265323244](https://github.com/dotnet/machinelearning/pull/2888#discussion_r265323244) [](ancestors = 265323244,265317859,265306118,264910410)